client/src/client.rs

```diff
@@ -166,6 +166,7 @@ impl ClientBuilder {
             data_dir,
             chain: base_config.chain,
             forks: base_config.forks,
+            checkpoint_duration: base_config.checkpoint_duration,
```

Comment:
can we rename this to something more descriptive, like `max_checkpoint_age`. Other than that LGTM.

Reply:
done